impf/browser.py

```diff
@@ -384,10 +385,7 @@ def control_main(self):
             self.logger.error('Maximum errors and retries exceeded - skipping location for now')
             return
 
-        # Quick Restart
-        if self.error_counter == 0: self.main_page()
```

Comment:
The error counter gives the bot some time to recover, instead of just ending it instantly

Reply:
We could make more use of `control_assert` which could try to recover. But doing it in `control_main` is not ideal because this will only be successful if the current page in the browser is `Wurde Ihr Anspruch auf eine Corona-Schutzimpfung bereits geprüft?` (`location_page`).